app/posts/2021-07-21-filter-updates.md

```diff
@@ -0,0 +1,92 @@
+
+## Hypothesis: Our users can’t locate the filters
+
+We had concerns around the discoverability of our filters on the search results screen. By default the filters were hidden within accordions. 
+
+{{ appFigure({
+  image: {
+    file: "original-filters.png",
+    alt: "A screenshot of our original filters"
+  },
+  caption: "How the filters looked in their closed and open state.."
+}) }}
+
+The GOV.UK design system [guidance on accordions](https://design-system.service.gov.uk/components/accordion/#when-not-to-use-this-component) states:
+
+>“Accordions hide content from users and not everyone will notice them or understand how they work … Accordions work best for simple content and links.”
+
+We wondered if exposing the filters at all times would improve visibility and usage.
+
+### What we did
+
+The updated filter design followed the visual pattern which had already been used on the Find postgraduate teacher training, Register trainee teachers and Manage teacher training applications services. We had confidence knowing these services were using a similar pattern and had been included in numerous rounds of usability testing. 
+
+Exposing the filters by default also represented a usability improvement, largely by removing the barriers to access them. When permanently exposed, there was no interaction required to open the filters before being able to use them. There was also far less chance a user wouldn't notice them.
+
+{{ appFigure({
+  image: {
+    file: "new-filters-desktop.png",
+    alt: "A screenshot of the updated filters at a wide viewport"
+  },
+  caption: "The updated filters at a wide viewport."
+}) }}
+
+{{ appFigure({
+  image: {
+    file: "new-filters.png",
+    alt: "A screenshot of the updated filters at a thin viewport"
+  },
+  caption: "The updated filters at a smaller viewport in its closed and open state."
+}) }}
+
+### How it went
+
+For the period of 11 October -  11 November 2021, approximately 18% of searches contained filtered results. 
+
+The use of filters when searching has significantly increased since the updates were made.
+
+## Hypothesis: We are encouraging searching again over filtering current results
+
+We also suspect we are encouraging a ‘search again’ behaviour (as opposed to filtering the current results) by placing the search form above the filters on the search results screen. 
+
+* Are we influencing our user's behaviour by placing the form in such a prominent position?
+* Has a bias towards searching unintentionally been introduced?
+* Is there a primary behaviour we want to encourage when viewing search results? Searching again or filtering?
+
+We have planned an A/B with a variant where the search and filter UI are decoupled. We are curious if this will affect search again frequency vs filter usage.
```

Comment:
```suggestion
We have planned an A/B test with a variant where the search and filter UI are decoupled. We are curious if this will affect search again frequency vs filter usage.
```